sdk/eventhubs/azure-messaging-eventhubs/pom.xml

```diff
@@ -37,7 +37,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.2.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.3.0-beta.2</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
```

Comment:
Just to make sure, there is an unreleased_com entry in the version_client.txt?

Reply:
There is https://github.com/Azure/azure-sdk-for-java/blob/master/eng/versioning/version_client.txt#L40